docs/ssr.mdx

```diff
@@ -120,7 +120,11 @@ res
 
 ```jsx
 // Hydration of the ids in `data-emotion-css` will automatically occur when the cache is created
```

Comment:
It's unrelated change but this part is also slightly incorrect and could be fixed together with the missing `key`
```suggestion
// Hydration of the ids in `data-emotion` will automatically occur when the cache is created
```

Reply:
Will I be able to squash (safe to quick merge that in) or should I amend my commit with that?

Reply:
I will squash this when merging to the main branch, it's fine to just commit this using the GitHub's UI button